### ISSUE TITLE
What should we benchmark?

### ISSUE THREAD

**Issue by ccbaumler** [dib-lab/genome-grist#263]
Following the same process as sourmash-bio/sourmash#2410, we will benchmark the genome-grist workflow with a combination of the six sequences listed in the sourmash-bio/sourmash#2410. My instinct is to:

1. Run each sequences alone
2. Run a variety of sequences from small to large sets
3. Run the all six together

We could also include benchmarking different databases across the steps above.

**Comment by ctb** [comment]
I don't think that genome-grist has any individually expensive steps or computationally complex scripts that are part of it; it's just the workflow overall that involves an awful lot of steps, much like charcoal. That may change your benchmarking strategy.
